src/bci_build/package/appcontainers.py

```diff
@@ -229,20 +237,24 @@ def _envsubst_pkg_name(os_version: OsVersion) -> str:
             for level in (ParseVersion.MAJOR, ParseVersion.MINOR, ParseVersion.PATCH)
         ],
         volumes=["/var/lib/prometheus"],
-        exposes_tcp=[9090],
+        exposes_tcp=[_PROMETHEUS_PORT],
+        custom_end=_generate_prometheus_family_healthcheck(
+            _PROMETHEUS_PORT, "Prometheus Server is Healthy."
```

Comment:
Do we really need to check for this specific string to be returned? from what I've seen in kubernetes manifests, it simply sets this as a livenessProbe, aka as long as the http return code is 200-399 it's good.

Reply:
@witekest what do you think about that suggestion? I'd prefer it, as it is simpler, but I'm not sure how the Prometheus stack behaves exactly

Reply:
Agree. The string is always the same and does not provide additional information.